index.html

```diff
@@ -59,6 +59,171 @@ <h2>I'm a software <br> developer</h2>
+                        <h3>Multi-Post Stories <br> Gain+Glory</h3>
+                        <ul>
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavScript</li>
+                            <li>Html</li>
+                        </ul>
+                        <button class="project-button" type="button">
+                            <a href="#">See Project</a>
+                        </button>
+                    </div>
+                </div>
+                <div class="cards">
+                    <div class="img-holder">
+                        <img src="img/Img Placeholder.png" alt="project6">
+                    </div>
+                    <div class="text-holder">
+                        <h3>Multi-Post Stories <br> Gain+Glory</h3>
+                        <ul>
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavScript</li>
+                            <li>Html</li>
+                        </ul>
+                        <button class="project-button" type="button">
+                            <a href="#">See Project</a>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </section>  
+         <!-- About me section -->
+         <section class="aboutme-section">
+            <div class="about-me">
+                <img src="img/Mask Group.png" alt="bg-img">
+                <h3>About me</h3>
+                <p class="flex"> I can help you build a product, <br> feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    <br> don’t hestiate to contact me.</p>
+                <button class="flex" type="button">
+                    <a href="#">Get My Resume</a>
+                </button>
+            </div>
+            <!-- Skills section -->
+            <div class="skills">
+                <div class="skills-card">
+                    <div class="icon1">
+                        <img src="img/icon1.png" alt="icon1">
+                    </div>
+                    <h3>Languages</h3>
+                    <ul class="flex">
+                        <li>JavScript</li>
+                        <li>Ruby</li>
+                        <li>Html</li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+                <div class="skills-card">
+                    <div class="icon1">
+                        <img src="img/icon2.png" alt="icon2">
+                    </div>
+                    <h3>Frameworks</h3>
+                    <ul class="flex">
+                        <li>Boottrap</li>
+                        <li>Ruby on rails</li>
+                        <li>RSpec</li>
+                        <li>Capybara</li>
+                        <li>Selenium</li>
+                    </ul>
```

Comment:
- [ ] Check this out, I would suggest you follow the design guidlines and update everything to match the correct figma template
<img width="626" alt="fram" src="https://user-images.githubusercontent.com/77843050/205413923-5fe8c9d8-80d5-47d7-8e17-c1d5e96f717b.png">